Backport nats-io/natscli 09d7a823 for Go 1.26 vet (log.Printf non-constant format, m.Reply method).

--- a/cli/pub_command.go
+++ b/cli/pub_command.go
@@ -193,7 +193,7 @@ func (c *pubCmd) writeAtomic(nc *nats.Conn) error {
 		if ack.Value != "" {
 			msg += fmt.Sprintf(" Counter Value: %s", ack.Value)
 		}
-		log.Printf(msg)
+		log.Print(msg)
 	}
 
 	return nil
@@ -339,7 +339,7 @@ func (c *pubCmd) doJetstream(nc *nats.Conn, pub *iu.Publisher) error {
 			if ack.Value != "" {
 				msg += fmt.Sprintf(" Counter Value: %s", ack.Value)
 			}
-			log.Printf(msg)
+			log.Print(msg)
 		}
 	}
 
--- a/cli/sub_command.go
+++ b/cli/sub_command.go
@@ -1,4 +1,4 @@
-// Copyright 2020-2025 The NATS Authors
+// Copyright 2020-2026 The NATS Authors
 // Licensed under the Apache License, Version 2.0 (the "License");
 // you may not use this file except in compliance with the License.
 // You may obtain a copy of the License at
@@ -466,7 +466,7 @@ func (c *subCmd) createJetStreamMsgHandler(subState subscriptionState, nc *nats.
 			defer func() {
 				err := m.Ack()
 				if err != nil && !subState.dump && !c.raw {
-					log.Printf("Acknowledging message via subject %s failed: %s\n", m.Reply, err)
+					log.Printf("Acknowledging message via subject %s failed: %s\n", m.Reply(), err)
 				}
 			}()
 		}
--- a/internal/bench/stats.go
+++ b/internal/bench/stats.go
@@ -1,3 +1,16 @@
+// Copyright 2026 The NATS Authors
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
 package bench
 
 import (
@@ -112,7 +125,6 @@ func (bm *BenchmarkResults) Report() string {
 
 	if len(bm.SampleGroup.samples) == 1 {
 		buffer.WriteString(fmt.Sprintf("%s %s stats: %s\n", bm.Name, GetBenchTypeLabel(bm.BenchType), bm))
-		indent += " "
 	} else {
